### PR TITLE
fix!: update partial / incomplete support for the SIMD proposal

### PR DESCRIPTION
**Pull request by bmeurer**
This imports the latest tests from the SIMD proposal and updates
the parser and the disassembler to properly support all operators.

Bug: https://crbug.com/1141798
Doc: https://github.com/WebAssembly/simd

### CODE REVIEW

File: src/WasmDis.ts [hunk 1]
```diff
@@ -132,6 +132,16 @@ function memoryAddressToString(
   var defaultAlignFlags;
   switch (code) {
     case OperatorCode.v128_load:
+      case OperatorCode.i16x8_load8x8_s:
```
Comment (ktran) [inline src/WasmDis.ts:135]: nit: indentation

**Review by ktran — APPROVED** [review]
LGTM!

I just checked the [history](https://github.com/WebAssembly/simd/commits/master/proposals/simd/NewOpcodes.md) of the change to the new opcodes; it seems that lately there have been more additions: 

i32x4.dot_i16x8_s | 0xba
f32x4.pmin | 0xea | f64x2.pmin | 0xf6
f32x4.pmax | 0xeb | f64x2.pmax | 0xf7
v128.load32_zero | 0xfc
v128.load64_zero | 0xfd

File: src/WasmDis.ts (hunk 1 above)
Comment (bmeurer) [reply src/WasmDis.ts]: Fixed, thanks.

**Comment by bmeurer** [comment]
> I just checked the [history](https://github.com/WebAssembly/simd/commits/master/proposals/simd/NewOpcodes.md) of the change to the new opcodes; it seems that lately there have been more additions:
> 
> i32x4.dot_i16x8_s | 0xba
> f32x4.pmin | 0xea | f64x2.pmin | 0xf6
> f32x4.pmax | 0xeb | f64x2.pmax | 0xf7
> v128.load32_zero | 0xfc
> v128.load64_zero | 0xfd

Thanks for the hint. I was going by the spec document, looks like that's not generally up to date. I've added the new operators.

**Pull request merged**